### DISCUSSION TITLE
Class now found in while using dbal 3.1.3

### DISCUSSION THREAD

**Discussion by bhushan** [doctrine/doctrine2#9110] [Support Questions] [answered]
2.10.x branch : with `"doctrine/dbal": "3.1.3",` 

doesnt have following classes present,

```php
Doctrine\DBAL\Platforms\PostgreSQLPlatform

Doctrine\DBAL\Platforms\SQLServerPlatform
```

I thought of removing it but in `3.2` version its present, also in palsm files i saw 

```xml
                <!-- DBAL 3.2 forward compatibility -->
                <referencedClass name="Doctrine\DBAL\Platforms\PostgreSQLPlatform"/>
                <referencedClass name="Doctrine\DBAL\Platforms\SQLServerPlatform"/>
```

should we update composer dependency to `3.2.*` or remove code ? 

I can send the pull request to remove code or update dependecies and updated code.. what to do ?

**Comment by bhushan** [comment, marked as answer]
@derrabus please review when you have time .. thank you

**Reply by derrabus** [reply]
We should do neither, the current code is fine. It is compatible with DBAL 2.13, 3.1 and 3.2.

**Reply by bhushan** [reply]
but in code these classes are not found? are these scenarios handled ?

**Reply by derrabus** [reply]
Is there an _actual_ issue, you encounter with the classes not being found?

**Reply by bhushan** [reply]
not really, just checking code and found classes used which are not found in dbal in many of the places, but I think you are right.. 

quick discussed here, for better idea, thanx
